components/reddit/sources/new-comments-by-user/new-comments-by-user.js

```diff
@@ -0,0 +1,78 @@
+const common = require("../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-by-user",
+  name: "New comments by user",
+  description: "Emits an event each time a user posts a new comment.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    username: { propDefinition: [common.props.reddit, "username"] },
+    numberOfParents: {
+      type: "integer",
+      label: "Number of parents",
+      description:
+        "The emitted events will contain the new comment plus the parents of said comment up to the number indicated in this property.",
+      default: 2,
+      min: 2,
+      max: 10,
+    },
+    timeFilter: { propDefinition: [common.props.reddit, "timeFilter"] },
```

Comment:
I'd recommend making this default in the app file (separately I mentioned you could override the `optional: true` in the specific component, but now that I see you're using it a couple of places, so ahead and set it to be optional in the app file.

Reply:
As in the other conversation, I set this optional with the default value in the app file.